File: Domain/Item.cs
namespace Domain
{
    public class Item
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string Category { get; set; }
        public string Description { get; set; }
        public string Details { get; set; }
        public List<string> Sizes { get; set; }
        public List<string> Colors { get; set; }
        public List<string> Genders { get; set; }
        public List<string> CompressionClasses { get; set; }
        public List<Photo> Images { get; set; }
        public double Price { get; set; }
        public double PriceSale { get; set; }
        public ICollection<Like> UserLikes { get; set; } = new List<Like>();
    }
}

File: client/src/models/Item.ts
export interface Item {
  id: string;
  name: string;
  description: string;
  details: string;
  sizes: string[];
  colors: string[];
  genders: string[];
  compressionClasses: string[];
  category: Category;
  images: string[];
  price: number;
  priceSale: number;
  added: boolean;
  favorite: boolean;
  leftCount: number;
}

export interface ItemOptions {
  size: string;
  color: string;
  gender: string;
  compressionClass: string;
}

export enum Category {
  corset = "Abdominal Corsets",
  bandage_foot = "Ankle and Foot Bandages",
  bandage_shoulder = "Bandages for Shoulder and Elbow Joints",
  bandage_arm = "Arm Bandages and Splints",
  child = "Children's Orthopedic Products",
  bandage_elastic = "Elastic Bandages and Other Products",
  nexus = "Nexus Adjustable Compression Products",
}
